content/webapp/components/IIIFItem/IIIFItem.AudioVideo.tsx

```diff
@@ -0,0 +1,89 @@
+import { play } from '@weco/common/icons';
+import { formatDuration } from '@weco/common/utils/format-date';
+import Button from '@weco/common/views/components/Buttons';
+import { useUser } from '@weco/common/views/components/UserProvider';
+import Download from '@weco/content/components/Download';
+import {
+  DownloadOption,
+  TransformedCanvas,
+} from '@weco/content/types/manifest';
+import {
+  getCanvasPaintingItem,
+  getVideoAudioDownloadOptions,
+  isAudioCanvas,
+} from '@weco/content/utils/iiif/v3';
+import { getAudioVideoLabel } from '@weco/content/utils/works';
+
+const getTypeAndDuration = (
+  canvas: TransformedCanvas
+): {
+  type: string;
+  duration?: string;
+} => {
+  const paintingItem = getCanvasPaintingItem(canvas);
+
+  const duration =
+    paintingItem &&
+    'duration' in paintingItem &&
+    typeof paintingItem.duration === 'number'
+      ? formatDuration(Math.round(paintingItem.duration))
+      : undefined;
+
+  const isAudio = isAudioCanvas(paintingItem);
+
+  return { type: isAudio ? 'Audio' : 'Video', duration };
+};
+
+const IIIFItemAudioVideo = ({ canvas, item, i, itemUrl, isRestricted }) => {
```

Comment:
np; could this be called IIIFItemAudioVideoLink or something similar. I'm liable to get it confused with the audio/video players

Reply:
Yes that makes sense